cartridge-cli.lua

```diff
@@ -1230,47 +1163,63 @@ local function form_distribution_dir(source_dir, dest_dir)
                  "normally ignored are shipped to the resulting package. ")
     end
 
-    -- deleting files matching patterns from .cartridge.ignore
-    remove_ignored(dest_dir)
-
+    info('Remove .git directory')
     remove_by_path(fio.pathjoin(dest_dir, '.git'))
-    remove_by_path(fio.pathjoin(dest_dir, '.cartridge.ignore'))
 
     -- check application files mode
+    info('Check application file modes')
     check_filemodes(dest_dir)
 end
 
 local function build_application(dir)
-    if fio.path.exists(fio.pathjoin(dir, '.cartridge.pre')) then
-        info("Running .cartridge.pre")
+    -- pre build
+    if fio.path.exists(fio.pathjoin(dir, PREBUILD_SCRIPT_NAME)) then
```

Comment:
This logic is too similar to `post build`... May be it's a good idea to move it into "run_hook" function